src/templates/index.html

```diff
@@ -11,26 +11,61 @@
     <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/styles.css') }}">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.13.5/xlsx.full.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.13.5/jszip.js"></script>
 </head>
 
 <body>
     <header>
         <div class="centered-content">
             <!-- Place your logo image here -->
             <img src="{{ url_for('static', filename='png/official_logo_de.png') }}" alt="Logo" class="logo">
-        </div>
+        </div>   
+        
+        <!--div id="popup" style="color: blue;">
+        <script src="https://translate.google.com/translate_a/element.js?cb=loadGoogleTranslate"></script>
+            <script> 
+            function loadGoogleTranslate()
+            {
+                    new google.translate.TranslateElement({pageLanguage: 'en' , includedLanguages : 'de,en',layout: google.translate.TranslateElement.InlineLayout.SIMPLE},'popup')
     
+            }
+            </script>
+        </div-->
+        <!--button id="myButton" type="button" >Deutsch</button>
+
+        <script type="text/javascript">
+    document.getElementById("myButton").onclick = function () {
+        location.href = "https://www.google.com";
+    };
+         </script-->
+        
+
         <!-- Container for the smaller logos on the top right -->
         <div class="small-logos-container" style="left:0%;right:100%">
             <img src="{{ url_for('static', filename='png/Helm_AI.png') }}"
             alt="Small Logo 1" class="custom-logo1">
             <img src="{{ url_for('static', filename='png/l2_MCML_Logo.jpeg') }}" alt="Small Logo 2" class="custom-logo1">
-            <img src="{{ url_for('static', filename='png/banner.png') }}" alt="Small Logo 1" class="custom-logo3">
+            <img src="{{ url_for('static', filename='png/banner.png') }}" alt="Small Logo 1" class="custom-logo1">
         </div>
-        <div class="small-logos-container">
+        <div id="popup"class="small-logos-container">
             <!-- Place your smaller logo images here -->
+            <script src="https://translate.google.com/translate_a/element.js?cb=loadGoogleTranslate"></script>
+            <script type="text/javascript"> 
+            function loadGoogleTranslate()
+            {
+                /*setCookie("googtrans", '/en/de', 1);*/   
+                new google.translate.TranslateElement({pageLanguage: 'en' , includedLanguages : 'de,en',layout: google.translate.TranslateElement.InlineLayout.SIMPLE},'popup')
+    
+            }
+            /*function setCookie(key, value, expiry) {
+                var expires = new Date();
+                expires.setTime(expires.getTime() + (expiry * 24 * 60 * 60 * 1000));
+                document.cookie = key + '=' + value + ';expires=' + expires.toUTCString();
+            }*/
+            </script>
```

Comment:
Are these cookies essential for having the translate function work at all? or just to remember the user's past language preferences? If they are necessary for it to work, we need to have a cookie consent banner, and a privacy policy that explains the specific cookie we use ("googtrans"). This is because this cookie isn't strictly necessary to use the website (see https://gdpr.eu/cookies/)

It would be ideal if we don't need to use this cookie. If we do, let's add a footer that explain that by continuing to use the website, the user agrees to the user of the google translate cookie and provide a link to the cookie's info.

Reply:
Template text for such a disclaimer (suggest do it in a tooltip or a dismissable banner (not a popup, because popups can be blocked).

Main issue is not sure how to go about making a privacy policy that works for our case. would need to research. 

This website may use cookies to enhance your browsing experience and provide personalized content. By continuing to use this site, you agree to the use of cookies as described in our [Privacy Policy](https://www.google.com). Cookies are small text files that are stored on your device. They may include anonymous identifiers that help us understand how you interact with our website.

Please note that by using this website, you consent to the use of cookies. If you do not wish to accept cookies, you may disable them in your browser settings. However, please be aware that some features of the website may not function correctly without the use of cookies.

For more information about how we handle your data and use cookies, please refer to our [Privacy Policy](https://www.google.com) linked at the bottom of the page.